### DISCUSSION TITLE
Widlerness notifiers for players who are in your combat range

### DISCUSSION THREAD

**Discussion by johnyuki** [runelite/runelite#16630] [Ideas]
I'm sick of panic tp'ing to players who can't actually attack me. It would be cool if there was an option that colour codes players overhead names based on whether they are in your current combat range, so red = can attack you and green = cannot attack you.

**Comment by Nightfirecat** [comment]
We cannot add that [per Jagex's Third Party Client Guidelines:](https://secure.runescape.com/m=news/third-party-client-guidelines?oldschool=1)

> Offers additional information about other players, with the purpose of scouting PvP targets
